modules/seo-tools/jetpack-seo-posts.php

```diff
@@ -60,4 +60,36 @@ public static function get_post_custom_description( $post ) {
 
 		return $custom_description;
 	}
-}
+
+	/**
+	 * Registers the self::DESCRIPTION_META_KEY post_meta for use in the REST API.
+	 */
+	public static function register_post_meta() {
+		if ( ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+			return;
+		}
+
+		$args = array(
+			'type' => 'string',
+			'description' => __( 'Custom post description to be used in HTML <meta /> tag.', 'jetpack' ),
+			'single' => true,
+			'default' => '',
+			'show_in_rest' => array(
+				'name' => self::DESCRIPTION_META_KEY
+			),
```

Comment:
Should we use the `sanitize_callback` parameter to make sure the meta that is saved is clean (no HTML) and no longer than 160 characters?

Reply:
Why 160 characters?

Reply:
We should not drop at 160, only warn the user that they've reached the optimal length. Search engines will parse longer meta descriptions and occasionally display them longer (although rare). I don't remember what the original length was (maybe 300?), but 300 would be a good spot.

As for sanitizing out HTML, yes, please. :)